src/lib/lsp-factory.ts

```diff
@@ -15,7 +15,7 @@ export class LSPFactory {
    * import { LSPFactory} from 'lspFactory.js';
    *
    * const myLSPFactory = new LSPFactory('0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266', provider);
-   * const { erc725Account, basicKeyManager } = await myLSPFactory.createLSP3UniversalProfile({
+   * const { erc725Account, keyManager } = await myLSPFactory.createLSP3UniversalProfile({
```

Comment:
I think you should use something which has the length of a private key for the example.

instead of using: `0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266`

It is misleading to see a public address at line 17 instead of a 64char long.
One could think we need to use the controllerAddress to instantiate the LSPFactory.

In a test bellow you use: `0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80`
And in the doc I read: `0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266`

Reply:
Thanks for spotting, I'll update that. 👍